src/jsoncharutils_tables.h

```diff
@@ -201,8 +201,8 @@ const uint32_t digit_to_val32[886] = {
 // per instruction set).
 ///
 
-constexpr int FASTFLOAT_SMALLEST_POWER = -344;
-constexpr int FASTFLOAT_LARGEST_POWER = 308;
+constexpr int simdjson_smallest_power = -344;
+constexpr int simdjson_largest_power = 308;
```

Comment:
Also, these don't need to be prefixed with simdjson_ because they are already in the simdjson namespace. Wonderful benefit of using constants instead of macros :)

Reply:
You are correct, they should not be prefixed.